test/test_tensor_creation_ops.py

```diff
@@ -716,7 +716,7 @@ def test_eye(self, device):
 
     @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
     @precisionOverride({torch.float: 1e-8, torch.double: 1e-10})
-    @dtypes(torch.float, torch.double)
+    @dtypes(*([torch.float, torch.double] + torch.testing.get_all_complex_dtypes()))
     def test_linspace_vs_numpy(self, device, dtype):
         start = -0.0316082797944545745849609375
```

Comment:
This should test complex values for the complex version of the test. Something like: `complex(-0.0316082797944545745849609375, .0<other long random string of digits>`